pkg/drivers/hyperkit/driver.go

```diff
@@ -121,7 +117,7 @@ func (d *Driver) Create() error {
 
 // DriverName returns the name of the driver
 func (d *Driver) DriverName() string {
-	return constants.DriverHyperkit
+	return "hyperkit"
```

Comment:
can we make it a constant inside the driver?

Reply:
yep, done.

Reply:
I disagree :-P